src/client/pages/RetroCertsAuthPage/index.js

```diff
@@ -48,12 +49,33 @@ function RetroCertsAuthPage(props) {
     </Row>
   );
 
+  function hasMissingFields() {
+    return validated && (!lastName || !eddcan || !ssn);
```

Comment:
Can we do some basic input fixing like removing leading/trailing whitespace from the fields? Maybe that could be a follow up change.

Reply:
I can do something for this when I go back in and update the 2 error messages you pointed out aboe.